src/partials/about.html

```diff
@@ -0,0 +1,21 @@
+<section id="how its made" class="container">
```

Comment:
Usuń id, jest niepotrzebny. 
Druga rzecz, container powienien być w divie otaczajacym cały kontent sekcji

Reply:
blok about możesz sobie zacząć odtąd

Reply:
o id napisalem ponizej ... "Przyciski pod tytułem “Products” i “How it’s made” to linki kotwiczne do odpowiednich sekcji.
"

Reply:
Masz, rację, przepraszam, id potrzebne :)